### ISSUE TITLE
Why were Maven builds removed?

### ISSUE THREAD

**Issue by khatchad** [wala/WALA#1262]
In addition to #1260 and 2fcbdee5c1980a75709d178afb6ff95434bbe46b that removed the OSGi components, I also noticed today that not only the discussion at https://github.com/wala/WALA/issues/742#issuecomment-606385878 is no longer happening, but the Maven builds were actually removed recently in b33d030fa546c119b8acad245f425d2935340fd5 and b92e667708d3e4853056a15e824a2561821412ee. That could explain the missing JAR dependencies I've noticed here: https://github.com/wala/WALA/issues/1260#issuecomment-1500499424.

Was this done to simply the recent subproject reorganization?

**Comment by msridhar** [comment]
@khatchad Maven builds were removed as keeping the Maven build working was a maintenance burden and much of the code built with Maven was unmaintained as far as we know.  For the Eclipse plugin code, the only way I know to test it properly is with Tycho, which doesn't play well with Gradle.

If you are reliant on the plugin-based code, I think the best way to revive it would be to move it outside the main WALA repository and restore the Maven build there.  At one point I looked into this and got stuck; see #462 for the work we did there.  Alternately, if there is some way to run Eclipse plugin tests atop Gradle, we could look into that.
